server/api/stats-api.js

```diff
@@ -30,6 +30,11 @@ module.exports = function(app, config) {
 
                     var allDeathGamesWithCause = stats.deathGamesWithCauses(allNormalModeGames);
 
+                    if(allDeathGamesWithCause.length == 0) {
```

Comment:
Thanks for the bugfix, can you add a test to `stats-api-test.js`?

Reply:
Done, and also fixed a bug discovered by another unit test.